network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/configuration/WgServerApi.kt

```diff
@@ -88,7 +87,7 @@ class RealWgServerApi @Inject constructor(
         serverName = server.name,
         publicKey = server.publicKey,
         publicEndpoint = server.extractPublicEndpoint(),
-        address = allowedIPs.joinToString(","),
+        address = allowedIPs.first(),
```

Comment:
@aitorvs will there never be a scenario that we will have more than 1 here?
If so, maybe better to pass the list?

Reply:
Tbh I am not sure what happens if we add more than one address to the `tun`. In principle it should just work but we'd need to test before.
Added the `first()` to avoid issues in the event that BE (wrongly) sends more than one address here.

This is actually a bit more convoluted because of why `allowsIPs` means `tun` address for the client. We can discuss offline